ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -14,16 +14,26 @@
 package com.github.ambry.config;
 
 import com.github.ambry.protocol.GetOption;
+import com.github.ambry.rest.RestMethod;
 import com.github.ambry.router.GetBlobOptions;
 import java.util.Arrays;
 import java.util.List;
+import org.json.JSONException;
+import org.json.JSONObject;
 
 
 /**
  * Configuration parameters required by the Ambry frontend.
  */
 public class FrontendConfig {
 
+  // Property keys
+  public static final String URL_SIGNER_ENDPOINTS = "frontend.url.signer.endpoints";
+
+  public static final String DEFAULT_ENDPOINT = "http://localhost:1174";
```

Comment:
hmm not sure that this should be `public` esp since you have getters in `AmbryUrlSigningService`

Reply:
Will change